README.md

```diff
@@ -1,227 +0,0 @@
-<img src="./images/hyf.svg" alt="image" width="200px" height="200px">
```

Comment:
You could get started with a nice README.md.

Could just be something like:

# Meal sharing app:
Developed as part `<homework>`

## Features:
- 1. feature
- 2. feature


## How to run